package/MDAnalysis/lib/distances.py

```diff
@@ -1523,3 +1526,16 @@ def apply_PBC(coords, box, backend="serial"):
         _run("triclinic_pbc", args=(coords, box), backend=backend)
 
     return coords
+
+def minimizing_vector(reference_point, ctrpos, box, backend="serial"):
```

Comment:
I am not sure the name is explicit enough as many things are vectors and minimizing may mean various things. `minimize_periodic_vector` perhaps?

Reply:
Will change it.